src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.TestLibrary/System.Text.Json.TestLibrary.Roslyn4.4.csproj

```diff
@@ -3,6 +3,6 @@
   <Import Project="System.Text.Json.TestLibrary.targets" />
 
   <ItemGroup>
-    <ProjectReference Include="..\..\gen\System.Text.Json.SourceGeneration.Roslyn4.0.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\..\gen\System.Text.Json.SourceGeneration.Roslyn4.4.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
```

Comment:
Same question here, don't you need the `SetTargetFramework="TargetFramework=netstandard2.0"` attribute? Without it, you are consuming the unshipping .NETCoreApp analyzer. I wonder why this even works today.